src/main/java/me/leoko/advancedban/utils/PunishmentType.java

```diff
@@ -11,6 +11,7 @@
     MUTE("Mute", null, false, "ab.mute.perma"),
     TEMP_MUTE("Tempmute", MUTE, true, "ab.mute.temp"),
     WARNING("Warn", null, false, "ab.warn.perma"),
+    NOTE("Note", null, false, "ab.Note"),
```

Comment:
I think the permission node would rather be something like "ab.note.use" according to our convention

Reply:
Will correct